src/Generator/Process/Overloaders.cs

```diff
@@ -672,6 +672,15 @@ public void WritePrologue(IndentedTextWriter writer, NameTable nameTable)
 
     public class GenCreateAndDeleteOverloader : IOverloader
     {
+        // Atm only Queries/Query needs this renaming
+        // - 2022-06-27
+        public static Dictionary<string, string> pluralNameToSingularName = new Dictionary<string, string>()
```

Comment:
This was the easier solution in my mind as this allows me to do a dictionary lookup instead of having to do a search through all of the postfixes that needs changing.

I'm not sure if there is a simple "constant time" lookup for postfixes here, so I just opted for having the entire name in the dictionary.